source/draw/gpu/opengl/AndroidContext.ooc

```diff
@@ -77,9 +77,10 @@ AndroidContext: class extends OpenGLContext {
 		sourcePointer := eglImage buffer lock(false)
 		length := channels * eglImage size width * eglImage size height
 		buffer := ByteBuffer new(sourcePointer, length,
-			func (buffer: ByteBuffer) {
+			func (b: ByteBuffer) {
 				eglImage buffer unlock()
 				this recyclePacker(gpuRgba)
+				b forceFree()
```

Comment:
I agree `buffer` wasn't a smart name here, @davidhesselbom isn't usually happy with names like `b` either but I'll let him decide for this case. Maybe `bytebuffer`.
Also - rock understands that you actually want a `_RecoverableByteBuffer`? That's nice.